### PR TITLE
Inductor cache clear

### PR DESCRIPTION
**Pull request by voznesenskym**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #94918

cc @soumith @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: torch/_inductor/codecache.py [hunk 1]
```diff
@@ -62,6 +62,11 @@ def cache_dir():
     )
 
 
+def remove_cache_dir():
```
Comment (voznesenskym) [inline torch/_inductor/codecache.py:65]: We could also keep the folder, and just clear the contents, but I kinda prefer this.

**Review by ezyang — APPROVED** [review]
I suggest adding a docblock

File: torch/_inductor/codecache.py [hunk 2]
```diff
@@ -62,6 +62,11 @@ def cache_dir():
...
+    import shutil
```
Comment (jansel) [inline torch/_inductor/codecache.py:66]: import at the top?

**Review by jansel — APPROVED** [review]

**Pull request closed without merging**